docs-2.0/nebula-dashboard-ent/4.cluster-operator/8.backup-and-restore.md

```diff
@@ -0,0 +1,104 @@
+# 使用 Nebula Dashboard 备份和恢复 Nebula Graph 数据
+
+为了防止操作失误或者系统故障导致数据丢失，我们需要经常对数据进行备份，Nebula Graph 提供 Backup & Restore（BR）工具可帮助用户备份和恢复图数据。Nebula Dashboard 企业版集成了 BR 能力并提供简单的 UI，用户只需几步即可进行数据备份和恢复操作。本文介绍如何使用 Nebula Dashboard 备份和恢复 Nebula Graph 数据。
+
+## 使用限制
+
+- 目前仅支持数据的全量备份和全量恢复，不支持增量备份和增量恢复。
+- 不支持图空间级别数据的备份和恢复。
+- 目前仅支持将数据备份至 Amazon S3 云存储，不支持备份至本地。 
+- 备份数据只能被恢复至原集群，不能跨集群恢复数据。
+- 不支持断点续传备份和恢复数据。
+- 目前仅支持查看备份和恢复操作产生的日志。
+
+## 前提条件
+
+- [使用 Dashboard 创建 Nebula Graph 集群](../3.create-import-dashboard/1.create-cluster.md)
+- [开通 Amazon S3 服务并创建存储桶](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html)
+
+## 操作步骤
+
+### 操作入口
+
+1. 在 Nebula Dashboard 企业版顶部导航栏，单击**集群管理**。
+2. 在目标集群右侧**详情**。
+3. 在左侧导航栏，单击**集群信息**->**备份恢复**。
+   
+### 备份数据
+
+用户通过创建备份文件的方式将备份的数据存至 Amazon S3 中，操作如下：
+
+1. 在**备份恢复**页面，单击**备份列表**。
+2. 在页面右上角，单击 **s3配置**。
+3. 填写 Amazon S3 云存储相关信息，并单击**提交**。
+
+  | 配置项        | 说明                                                         |
+  | ------------- | ------------------------------------------------------------ |
+  | s3.access_key | 访问 Amazon 服务的凭证，例如，`AKIAI44QH8DHBxxxx`。          |
+  | s3.endpoint   | Amazon S3 对外服务的访问域名 URL，例如`s3.us-east-2.amazonaws.com`。 |
```

Comment:
`https://{bucket_name}.s3.us-west-2.amazonaws.com` This format is not supported at present. You need to add bucket name to the storage path parameter